src/network.toit

```diff
@@ -195,6 +212,7 @@ class EndpointHttp implements Endpoint:
 
       // Handle installing containers.
       else if path == "/install" and request.method == "PUT":
+       request-mutex.do:
```

Comment:
The indentation is off, but that's because the same lines will have another refactoring in a follow-up PR, and this way there are fewer changes.